### PR TITLE
chore: lint tokenfactory

### PR DESCRIPTION
**Pull request by faddat**
## What is the purpose of the change

This PR lints the token factory. 

## Testing and Verifying

This change is a trivial rework / code cleanup without any test coverage.

## Documentation and Release Note

  - [ ] Does this pull request introduce a new feature or user-facing behavior changes? no
  - [ ] Changelog entry added to `Unreleased` section of `CHANGELOG.md`? no

Where is the change documented? 
  - [ ] Specification (`x/{module}/README.md`)
  - [ ] Osmosis documentation site
  - [ ] Code comments?
  - [x] N/A

### CODE REVIEW

**Review by pysel — APPROVED** [review]

File: x/tokenfactory/keeper/createdenom_test.go [hunk 1]
```diff
@@ -49,11 +49,11 @@ func (s *KeeperTestSuite) TestMsgCreateDenom() {
...
 	// Make sure that a second version of the same denom can't be recreated
-	res, err = s.msgServer.CreateDenom(sdk.WrapSDKContext(s.Ctx), types.NewMsgCreateDenom(s.TestAccs[0].String(), "bitcoin"))
+	_, err = s.msgServer.CreateDenom(sdk.WrapSDKContext(s.Ctx), types.NewMsgCreateDenom(s.TestAccs[0].String(), "bitcoin"))
 	s.Require().Error(err)
 
 	// Creating a second denom should work
-	res, err = s.msgServer.CreateDenom(sdk.WrapSDKContext(s.Ctx), types.NewMsgCreateDenom(s.TestAccs[0].String(), "litecoin"))
+	_, err = s.msgServer.CreateDenom(sdk.WrapSDKContext(s.Ctx), types.NewMsgCreateDenom(s.TestAccs[0].String(), "litecoin"))
 	s.Require().NoError(err)
 	s.Require().NotEmpty(res.GetNewTokenDenom())
```
Comment (p0mvn) [inline x/tokenfactory/keeper/createdenom_test.go:58]: This doesn't look right. We are using an old `res` variable on line 58 but the intention is to validate the updated one.

**Review by p0mvn — CHANGES_REQUESTED** [review]

File: x/tokenfactory/keeper/createdenom_test.go (hunk 1 above)
Comment (ValarDragon) [reply x/tokenfactory/keeper/createdenom_test.go:58]: If a linter is outputting this, we should not use this linter.

Comment (faddat) [reply x/tokenfactory/keeper/createdenom_test.go:58]: It was me, I'll take a look.  Thanks for your review. (this was not an automatically generated fix)

**Comment by faddat** [comment]
Hey, just wanted to let everyone know the result of this -- basically, I think that I made a mistake when using the linter.  It doesn't flag that line in any way.

**Review by p0mvn — APPROVED** [review]
LGTM, thanks

**Pull request merged**